programs/bpf/move_funds_c/src/move_funds.c

```diff
@@ -0,0 +1,129 @@
+
+//#include <stdint.h>
+//#include <stddef.h>
+
+#if 1
+// one way to define a helper function is with idex as a fixed value
```

Comment:
“idex” => “index”?

Reply:
Where does the 6 come from?

Reply:
Linux kernel definition.  This is another possible modification I'd like to make to rbpf to make it more general